<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>830</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>830</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cryptocurrency Tracker</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/pics/growth.png</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>830</width>
      <height>620</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/pics/cryptobackground_new2.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>381</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(230, 230, 230);</string>
    </property>
    <property name="text">
     <string>Choose Your Crypto:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 120);
border: 2px solid rgb(0, 54, 0);
border-radius: 10px;
color: rgb(0, 0, 0);
padding: 15px;</string>
    </property>
    <item>
     <property name="text">
      <string>BTC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ETH</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BNB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>XRP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>USDT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SOL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DOGE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ADA</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>260</y>
      <width>450</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 100);
border: 1px solid rgb(0, 54, 0);
border-radius: 10px;
color: rgb(0, 0, 0);
padding: 6px;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:15pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_crypto_bt">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>250</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.205056 rgba(4, 43, 0, 255), stop:1 rgba(12, 145, 0, 255));
	border: 1px solid rgb(0, 54, 0);
	border-radius: 10px;
	color: rgb(230, 230, 230);
}

QPushButton:hover{
	border: 4px solid rgb(0, 54, 0);
}

QPushButton:focus{
	border: 4px solid rgb(0, 54, 0);
}</string>
    </property>
    <property name="text">
     <string>Set Crypto</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
